dgbpy/hdf5.py

```diff
@@ -397,6 +397,12 @@ def validInfo( info ):
     return False
   return True
 
+def getText( infos, ky ):
+  ret = odhdf5.getAttr( infos, ky )
```

Comment:
I would place the logic differently:
if odhdf5.getIArray( infos, modeloutstr+'Size' ) > 1:
  return odhdf5.getText( info, ky )
else:
  return odhdf5.getAttr( info, ky )

Probably then not worth making it a dedicated function, you may place it directly on line 450